apps/site/pages/docs/api-extensions.mdx

```diff
@@ -219,4 +216,37 @@ query {
 }
 ```
 
+---
+
+## Extending Queries Using Fragments
+
+When developers need to include custom fields in existing server queries, they can utilize the src/fragments folder. Within this directory, they can generate files containing GraphQL fragments, which can then be integrated into the desired queries for extension purposes.
+
```

Comment:
```suggestion
To add additional fields to an existing query, follow the steps below. We will use the `ClientProductGallery` query as an example to demonstrate how to extend a query.

import { Callout } from 'nextra/components'
 
<Callout type="info" emoji="ℹ️">
  We will use the `ClientProductGallery` query to illustrate.
</Callout>

Step 1: Create a New File
Create a new `.ts` file in the `src/fragments` folder named `ClientProductGallery.ts`. *The name of the file should match the name of the query you want to extend.*

Step 2: Define the GraphQL Fragment
In the `ClientProductGallery.ts` file, define the GraphQL fragment corresponding to the new fields you want. In this example, the new field is represented by `newField` with a `data` property to retrieve. Use the following syntax as a guideline: 
```

Reply:
After `step 2`, the idea is to have the code introduced in the [230 line](https://github.com/vtex/faststore/pull/1943/files#diff-1814207badc0abee689ae7fa4154d83970f77ea8e797626c99515700e23803cdR230). Also, it would be helpful for whom is following the guide to have the result of this example or at least what should they expect from it.